channels/alpha

```diff
@@ -7,6 +7,8 @@ spec:
     - name: kope.io/k8s-1.5-debian-jessie-amd64-hvm-ebs-2017-01-09
       providerID: aws
       kubernetesVersion: ">=1.5.0"
+    - providerID: gce
+      name: "cos-cloud/cos-stable-56-9000-84-2"
```

Comment:
Noted. It does work if you set the source image url to `https://www.googleapis.com/compute/v1/projects/cos-cloud/global/images/cos-stable-57-9202-56-0`. I guess that answers my original question.